### ISSUE TITLE
[Suggestion, Compatibility] Option to change what torches villagers place

### ISSUE THREAD

**Issue by Answordy** [mactso/Regrowth#20]
Sorry if it was already discussed somewhere, I didn't find anything on this regard. But if you like the idea you could add an option to change what type of torch villagers place, mostly for mod compatibility with torch mods like Hardcore torches and Realistic torches.

**Comment by Atlantispy** [comment]
@mactso Someone already suggested this one!

**Comment by Atlantispy** [comment]
https://www.curseforge.com/minecraft/mc-mods/hardcore-torches-forge
https://www.curseforge.com/minecraft/mc-mods/realistic-torches

Links to the mentioned mods, as well as torches, they also add lanterns having oil which can be relit.

**Comment by mactso** [comment]
This is likely for the next release.

**Comment by weirdspecter** [comment]
Out of interest -- and I hate to be a bother -- was this ever implemented? And if so, was it implemented on any of the 1.19.2-compatible versions, or only on 1.19.3-onwards? Thanks.

**Comment by mactso** [comment]
No, I'm sorry but it was never implemented.     I didn't even remember it!   😞 

You are not being a bother.    This was a really rough last year tho with two stays in the hospital and two natural disasters including a week without electricity both times.

Then the newforge split.   

This is for Regrowth right?  To have them place a different kind of torch than standard torches?

**Comment by mactso** [comment]
I'm pinning this so it will appear at the top of my list of bug / questions .
